mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/services/BrukerService.kt

```diff
@@ -0,0 +1,30 @@
+package no.nav.mulighetsrommet.api.services
+
+import kotlinx.serialization.Serializable
+import no.nav.mulighetsrommet.api.clients.oppfolging.VeilarboppfolgingClient
+import no.nav.mulighetsrommet.api.clients.vedtak.VeilarbvedtaksstotteClient
+import no.nav.mulighetsrommet.api.domain.Innsatsgruppe
+
+class BrukerService(
+    private val veilarboppfolgingClient: VeilarboppfolgingClient,
+    private val veilarbvedtaksstotteClient: VeilarbvedtaksstotteClient
+) {
+
+    suspend fun hentBrukerdata(fnr: String, accessToken: String?): Brukerdata {
+        val oppfolgingsenhet = veilarboppfolgingClient.hentOppfolgingsstatus(fnr, accessToken)
+        val sisteVedtak = veilarbvedtaksstotteClient.hentSiste14AVedtak(fnr, accessToken)
+
+        return Brukerdata(
+            fnr = fnr,
+            oppfolgingsenhet = oppfolgingsenhet?.oppfolgingsenhet?.navn ?: "Oppfølgingsenhet ikke satt",
```

Comment:
Skal vi kaste en feil dersom oppfølgingsenhet ikke blir hentet? Uten en oppfølgingsenhet så vet vi ikke hvilke tiltaksgjennomføringer som skal vises for veileder. Vi kan ikke vise alle tiltaksgjennomføringer dersom vi ikke finner korrekt oppfølgingsenhet...